pricing.js

```diff
@@ -1,37 +1,49 @@
-module.exports.calculateProductPrice = function (product, employee, coverageLevels) {
+function calculateVolLifePrice(product, coverageLevels) {
   var price = 0
   var dollarsOff = 0
+  for (var i = 0; i < coverageLevels.length; i++) {
+    var coverageAmount = coverageLevels[i].coverage
+    price += (coverageAmount / product.cost.costDivisor) * product.cost.price
+  }
 
-  switch (product.type) {
-    case 'volLife':
-      for (var i = 0; i < coverageLevels.length; i++) {
-        var coverageAmount = coverageLevels[i].coverage
+  if (product.employerContribution.mode === 'dollar') {
```

Comment:
Notice that lines 9-15 are repeated on lines 27-32. In the interest of DRY (don't repeat yourself) code you would make this its own function as well.